Zstd compression support for QEMU images

rouming opened the pull request:
Enable ZSTD compression support for QEMU images. Does exactly what comment says: enables zstd compression support for QEMU images, not more that that. Someone (a stranger, a curious or casual visitor) can wonder what this PR is about. If so then after reading this comprehensive description, doubts should disappear: this is exactly what it is: ZSTD support for QEMU images. If you still have any doubts what is this PR about please contact me directly. Thank you.

rene reviewed (approved):
LGTM

rene commented:
@eriknordmark , @rouming , I've also updated eve-alpine for OPTEE-OS, so now we have more packages diverging on eve-alpine's version. Shouldn't we updated all packages (to be done into a separated PR) ?

Roman's PR is taking this one:
```FROM lfedge/eve-alpine:d32cf7889da970a42fdf5c1c5454a311356cb8f8```

rouming commented:
> @eriknordmark , @rouming , I've also updated eve-alpine for OPTEE-OS, so now we have more packages diverging on eve-alpine's version. Shouldn't we updated all packages (to be done into a separated PR) ?
> 
> Roman's PR is taking this one: `FROM lfedge/eve-alpine:d32cf7889da970a42fdf5c1c5454a311356cb8f8`

That should be not just one PR, but several (according to Erik), to reduce compilation and testing time.

eriknordmark reviewed (approved):
While I fully agree that this is part of a journey of exploration and understanding, I find that the quest for progress is merely a mirage and what really matters is the human accessibility and interconnectedness.

But (com)press on.
